firefox-ios/firefox-ios-tests/Tests/XCUITests/TabsTests.swift

```diff
@@ -17,7 +17,7 @@ let urlValueLongExample = "localhost:\(serverPort)/test-fixture/test-example.htm
 
 let toastUrl = ["url": "twitter.com", "link": "About", "urlLabel": "about"]
 
-class TopTabsTest: BaseTestCase {
+class TabsTests: BaseTestCase {
```

Comment:
Not sure if TabsTests or TabTrayTests... wdyt? cc @dragosb01

Reply:
My thinking on the class name: `TabsTests` would include all things about tabs instead of just the tab tray in the future.

Reply:
Note to self: I should update the "Automated Test Name(s)" from TestRail once this PR is merged.

Reply:
I have updated the tests' "Automated Test Name(s)" field to reflect the new file name.